src/main/java/com/ahsoka/SALC/user_model/controller/EmailValidator.java

```diff
@@ -1,5 +1,29 @@
 package com.ahsoka.SALC.user_model.controller;
 
-public class EmailValidator {
-	
+import lombok.NoArgsConstructor;
+
+import java.util.function.Predicate;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@NoArgsConstructor
+public class EmailValidator implements Predicate<String> {
+
+    Matcher matcher;
+    private static final String EMAIL_REGEX = "^(.+)@salc.org";
+    Pattern pattern = Pattern.compile(EMAIL_REGEX);
+
+    @Override
+    public boolean test(String email) {
+
+        matcher = pattern.matcher(email);
+        if (matcher.matches())
```

Comment:
¿Para qué escribir 8 líneas de código, pudiendo dejarlo en 1?
```suggestion
        return matcher.matches();
```

Reply:
Pues para hacerlo más visual para alguien que ve el código pero la verdad es que no tiene sentido alguno.